doc/book/zend.authentication.adapter.dbtable.md

```diff
@@ -0,0 +1,304 @@
+# Database Table Authentication
+
+> **note**
+>
+> `Zend\Authentication\Adapter\DbTable` has been deprecated, as its responsibilities have been
+splitted off into `Zend\Authentication\Adapter\DbTable\CallbackCheck` and
+`Zend\Authentication\Adapter\DbTable\CredentialTreatmentAdapter`. Use
+`Zend\Authentication\Adapter\DbTable\CredentialTreatmentAdapter` instead of
+`Zend\Authentication\Adapter\DbTable`.
```

Comment:
Each of the above lines needs to be prefixed with `> ` to be part of the blockquote for the note.